tentacle/Cargo.toml

```diff
@@ -25,6 +25,8 @@ tokio-util = { version = "0.6.0", features = ["codec"] }
 log = "0.4"
 bytes = "1.0.0"
 thiserror = "1.0"
+once_cell = "1.0"
+parking_lot = { version = "0.11", optional = true }
```

Comment:
need to add a CI task to enable this feature test and clippy

Reply:
add a feature check ci，since the interface is consistent with std, clippy check is not added